src/components/print/PrintLayerService.js

```diff
@@ -39,12 +39,12 @@ goog.require('ga_urlutils_service');
   });
 
   var pdfLegendString = '_big.pdf';
-  var POINTS_PER_INCH = 72; //PostScript points 1/72"
+  var POINTS_PER_INCH = 72; // PostScript points 1/72"
   var UNITS_RATIO = 39.37; // inches per meter
```

Comment:
Not related to this PR: We have the same variables in the directives. Maybe define them in this service and expose them from here to be used in the directive.